code/Controllers/CMSPageHistoryController.php

```diff
@@ -159,7 +159,7 @@ public function EditForm($request = null)
      * Returns the read only version of the edit form. Detaches all {@link FormAction}
      * instances attached since only action relates to revert.
      *
-     * Permission checking is done at the {@link CMSMain::getEditForm()} level.
+     * Permission checking is done at the {@link SilverStripe\CMS\Controllers\CMSMain::getEditForm()} level.
```

Comment:
yeah, that's my concern that if we end up adding a `use` later on, then the Docs are out of date using the FQN when not needed...

I'll leave the change as optional, but nice-to-have :)

@tractorcow mentions:
> please leave them short, and add a `use` if needed
> we don't need unreadable PHPDoc